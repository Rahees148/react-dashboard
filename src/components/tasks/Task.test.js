
import renderer from 'react-test-renderer';
import Task from './Tasks';


it('renders correctly', () => {
  const tree = renderer
    .create(<Task />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});