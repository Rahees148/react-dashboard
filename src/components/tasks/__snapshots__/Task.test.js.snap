// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="sc-gsTCUz fmULOU"
  >
    <h3
      className="sc-iqHYGH jNIVzu"
    >
      Tasks
    </h3>
    <div
      className="sc-dlfnbm jiPmFS"
    >
      <input
        className="sc-hKgILt gpIyHs"
        onChange={[Function]}
        placeholder="Search by task name"
        type="search"
      />
      <button
        className="sc-kEjbxe khqoZl"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        + New Task
      </button>
    </div>
  </div>,
  <div
    className="sc-bdfBwQ iUEVFm"
  >
    <div>
      No Task Found with 
      
       
    </div>
  </div>,
]
`;
